src/main/java/com/project/workingtime/controller/CheckerController.java

```diff
@@ -0,0 +1,89 @@
+package com.project.workingtime.controller;
+
+import com.project.workingtime.repository.Checker;
+import com.project.workingtime.repository.CheckerRepository;
+import com.project.workingtime.service.CheckerService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+public class CheckerController {
+
+    @Autowired
+    private CheckerRepository repository;
+
+    @RequestMapping(value="/check-in", method= RequestMethod.POST)
+    public String checkIn() {
+
+        // Take the last checker
+        Long id = repository.count();
+        Optional<Checker> checker = repository.findById(id);
+
+        if(checker.isPresent()) {
+
+            Checker lastChecker = checker.get();
+            LocalDateTime currentDateTime = LocalDateTime.now();
+            LocalDateTime lastCheckerDateTime = CheckerService.stringToLocalDateTime(lastChecker.getCheckin());
```

Comment:
All your business rules are in the controller but they should be on the service level.